src/test/java/com/flowingcode/vaadin/addons/localecombobox/LocaleComboBoxDemo.java

```diff
@@ -21,17 +21,71 @@
 package com.flowingcode.vaadin.addons.localecombobox;
 
 import com.flowingcode.vaadin.addons.demo.DemoSource;
+import com.vaadin.flow.component.Text;
 import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.orderedlayout.FlexComponent.Alignment;
+import com.vaadin.flow.component.orderedlayout.FlexComponent.JustifyContentMode;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
 import com.vaadin.flow.router.PageTitle;
 import com.vaadin.flow.router.Route;
+import java.util.Locale;
 
 @DemoSource
-@PageTitle("LocaleComboBox Add-on Demo")
+@PageTitle("Display modes")
 @SuppressWarnings("serial")
 @Route(value = "demo", layout = LocaleComboBoxDemoView.class)
 public class LocaleComboBoxDemo extends Div {
 
   public LocaleComboBoxDemo() {
-    add(new LocaleComboBox());
+
+    LocaleComboBox defaultDisplayLocale = new LocaleComboBox();
+    LocaleComboBox franceLocaleCombo = new LocaleComboBox();
+    LocaleComboBox germanyLocaleCombo = new LocaleComboBox();
+    LocaleComboBox koreanLocaleCombo = new LocaleComboBox();
+    LocaleComboBox selectedLocaleCombo = new LocaleComboBox();
+
+    defaultDisplayLocale.setValue(Locale.ITALY);
+    
+    franceLocaleCombo.setDisplayLocale(Locale.FRANCE);
+    franceLocaleCombo.setDisplayMode(LocaleComboBox.DISPLAY_CUSTOM);
+    franceLocaleCombo.setValue(Locale.ITALY);
+
+    germanyLocaleCombo.setDisplayLocale(Locale.GERMANY);
```

Comment:
Maybe with just one LocaleComboBox.DISPLAY_CUSTOM example is enough, let's remove the extra ones